finality-aleph/src/network/split.rs

```diff
@@ -114,9 +129,18 @@ impl<LeftData: Data, RightData: Data, R: ReceiverComponent<Split<LeftData, Right
 {
     async fn next(&mut self) -> Option<RightData> {
         loop {
+            trace!(target: "aleph-network", "Right Receiver Next Loop started a next iteration");
```

Comment:
Especially this one, since it's asymmetric. :P

Reply:
Done

Reply:
Doesn't seem to be.